File: sdkwork-app-sdk-csharp/Models/AuthorInfo.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class AuthorInfo
    {
        public int? Id { get; set; }
        public string? Name { get; set; }
        public string? Avatar { get; set; }
        public string? Bio { get; set; }
        public bool? IsFollowing { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/author-info.ts
/** 作者信息 */
export interface AuthorInfo {
  /** 作者ID */
  id?: number;
  /** 作者名称 */
  name?: string;
  /** 作者头像 */
  avatar?: string;
  /** 作者简介 */
  bio?: string;
  /** 是否关注 */
  isFollowing?: boolean;
}
